lib/p2p/Pool.ts

```diff
@@ -17,10 +17,18 @@ import assert from 'assert';
 import { ReputationEvent } from '../types/enums';
 
 type PoolConfig = {
+  /** Whether or not to automatically detect and share current external ip address on startup. */
+  detectexternalip: boolean;
+
+  /** If false, don't send GET_NODES when connecting, defaults to false. */
```

Comment:
Is the default true or false?
it is true in config.ts line 62 but comment here says that the default is false.

Reply:
Oh sorry my bad, fixing the comment